### PR TITLE
refactor: Migrate Edgex compose to 3.0

### LINKED ISSUE #121: Create new EdgeX 3.0 compose file
Best to use [Minnesota compose builder](https://github.com/edgexfoundry/edgex-compose/tree/minnesota/compose-builder) to create new edgex compose file.

Run `make gen ds-mqtt mqtt-bus no-secty` and then remove the EdgeX service that are not needed for AC like `Support-Scheduler`, `App-Rules-Engine`, etc. from the generated `docker-compose.yml`

Add volume mounted res folder for Device MQTT
```
    volumes:
      - ./res/device-mqtt:/res
```
Remove https://github.com/intel-retail/automated-vending/blob/main/res/mqtt/mosquitto.conf it is not used by existing compose file.

### PR DESCRIPTION
**Pull request by lenny-intel**
closes #121

## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-vending/blob/main/.github/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes **N/A**
- [ ] Updated Documentation as relevant to the changes **N/A**
- [x] All commented out code has been removed 
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency. **N/A**
- [ ] PR change contains code related to security **N/A**
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below) **N/A**

## What are you changing?
Migrated the EdgeX compose file to EdgeX 3.0 services

## Issue this PR will close

close: #**121**

## Test Instructions if applicable
Run `make run-edgex`
Verify all service, except device-mqtt, bootstrap without error

### CODE REVIEW

**Review by brian-intel — APPROVED** [review]

**Review by seanohair22 — APPROVED** [review]
LGTM! EdgeX Services spin up properly!

File: Makefile [hunk 1]
```diff
@@ -39,6 +39,9 @@ run-portainer:
 run:
```
Comment (ejlee3) [inline Makefile:39]: do we need to expand the copyright at the top to 2023?

**Review by ejlee3 — CHANGES_REQUESTED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (seanohair22) [reply Makefile:39]: The copyright can be adjusted and standardized in a follow-up PR

**Pull request merged**